### PR TITLE
THRIFT-4733: Fix test unix socket address already in use

### PR DESCRIPTION
**Pull request by Junnplus**

### CODE REVIEW

File: lib/py/test/test_sslsocket.py [hunk 1]
```diff
@@ -213,11 +213,9 @@ def test_unix_domain_socket(self):
             return
         fd, path = tempfile.mkstemp()
         os.close(fd)
-        try:
-            server = self._server_socket(unix_socket=path, keyfile=SERVER_KEY, certfile=SERVER_CERT)
-            self._assert_connection_success(server, path=path, cert_reqs=ssl.CERT_NONE)
-        finally:
-            os.unlink(path)
```
Comment (jeking3) [inline lib/py/test/test_sslsocket.py]: We should still be removing it after the test (finally).  It makes sense to remove it before as you have added.

Comment (Junnplus) [reply lib/py/test/test_sslsocket.py]: you're right.

**Comment by jeking3** [comment]
It looks like a real issue - empty commit to kick the build won't change the outcome.

**Comment by jeking3** [comment]
Please rebase on master as the issue seen in these builds was resolved.

**Pull request merged**